test.sh

```diff
@@ -1,51 +1,99 @@
 #!/bin/bash -e
-# test suite...
-echo running test.sh
-echo "ENV:"
-env
+# runs all (or selected) test suite(s) in test/ and aggregates results
+# Usage:
+# 	./test.sh
+#   ./test.sh gofmt
```

Comment:
In general, I've avoided places where I had more than one space, unless explicitly necessary. Please switch to tabs if you want indentation.

Reply:
i'm a python dev so that will take some getting used to ;), but willdo :)